osf/migrations/0084_preprint_node_divorce.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-03-12 18:25
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import transaction
+
+def divorce_preprints_from_nodes(apps, schema_editor):
+    Preprint = apps.get_model('osf', 'PreprintService')
+    PreprintContributor = apps.get_model('osf', 'PreprintContributor')
+    # tried to use F() function here but F() doesn't support table joins
+    # instead, using the following to make this transaction atomic
+    with transaction.atomic():
+        for preprint in Preprint.objects.filter(node__isnull=False):
+            preprint.title = preprint.node.title
+            preprint.description = preprint.node.description
+            preprint.creator = preprint.node.creator
```

Comment:
It might make sense to add a `migrated` DateTimeField to mark the time when a preprint was migrated. This will allow us to distinguish "legacy" preprints, which will have some expected differences from new preprints (legacy preprints won't have back-filled logs, for example).